playground/elements/nextjs/playwright.config.ts

```diff
@@ -0,0 +1,58 @@
+import { defineConfig, devices } from '@playwright/test'
+import path from 'path'
+
+const PORT = process.env.PORT || 3000
+const baseURL = `http://localhost:${PORT}`
+
+// Reference: https://playwright.dev/docs/test-configuration
+export default defineConfig({
```

Comment:
This should live inside the `integration` folder, not inside a playground app.

Looking at the playground app, it doesn't seem super specific to Clerk Elements and don't think it will. So I'd say let's remove this playground and use the `next-app-router` template of our integration test suite and use `.addFile()` to add any missing files